docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-SOAPCALL.xml

```diff
@@ -69,7 +69,7 @@
             form usernames and passwords, if required. If calling an ESP Web
             service, you can append the ver_=n.nn parameter to specify the
             version of the service. For example: <programlisting
-            language="ECL" role="notrunnable">SOAPCALL('http://127.0.0.1:8010/Wsdfu/?ver_=1.22', 
+            language="ECL" role="notrunnable">SOAPCALL('http://service.example.com:8010/Wsdfu/?ver_=1.22', 
```

Comment:
@JamesDeFabia 
Here I would use https and instead of service I would use "eclwatch".

'https://eclwatch.example.com:8010/Wsdfu/?ver_=1.22'

Reply:
@afishbeck  made this change, back to you